index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="initial-scale=1, width=device-width" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" />
+  </head>
+  <body>
+    <header class="header">
+      <img class="logo" src="./logo.png" alt="logo">
+        <ul class="navbar">
+          <li class="nav-icon">Home</li>
+          <li class="nav-icon">My Projects</li>
+          <li class="nav-icon">Contact US</li>
+        </ul>
+    </header>
+    
+    <section class="main container">
+      <img class="mobile-icon" src="./public/headershapes-mobile.svg" />
+      <div class="headline">
+        <b class="primary-text">
+          <span>
+            <p class="im-qaisar">I’m Qaisar.</p>
+            <p class="im-qaisar">Glad to see you!</p>
+          </span>
+        </b>
+        <div class="primary-text1">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hesitate to contact
+          me.
+        </div>
+        <div class="social-media">
+          <div class="supporting-text">
+            <div class="lets-connect">LET’S CONNECT</div>
+        </div>
+      </div>
+        <img class="social-media-icons" src="./public/social-media-icons.svg" />
```

Comment:
- [ ] Instead of using an image to display all social icons, you should create a list & group them using a `<ul>` then store each icon inside a `li`, separately. Each icon should also be stored inside a `<a>` tag. Since those are links to your social accounts.